cmd/cosign/policy_webhook/main.go

```diff
@@ -34,6 +34,14 @@ import (
 
 	"github.com/sigstore/cosign/pkg/apis/cosigned/v1alpha1"
 	"github.com/sigstore/cosign/pkg/reconciler/clusterimagepolicy"
+
+	// Register the provider-specific plugins
+	_ "github.com/sigstore/sigstore/pkg/signature/kms/aws"
+	_ "github.com/sigstore/sigstore/pkg/signature/kms/azure"
+	_ "github.com/sigstore/sigstore/pkg/signature/kms/gcp"
+	_ "github.com/sigstore/sigstore/pkg/signature/kms/hashivault"
+	// TODO(mattmoor): Not sure why this is in the final binary...
+	// _ "github.com/sigstore/sigstore/pkg/signature/kms/fake"
```

Comment:
`fake` was added for testing, it should just be required in a few test files

Reply:
Yeah, I suspected that, but it was linked from `pkg/reconciler/...` (not in test code!), so I wanted to run this through CI to figure out what tests are expecting this.  I'm going to guess now that this was bad copy/pasta and just remove this instance.

Reply:
This is removed in the commit I just pushed.